src/main/java/com/xceptance/xlt/report/util/JFreeChartUtils.java

```diff
@@ -987,9 +993,24 @@ public static void createPlaceholderChart(final File outputDir, final int width,
             graphics.drawString(XltConstants.REPORT_CHART_PLACEHOLDER_MESSAGE, (width - stringWidth) / 2, height / 2 + stringHeight / 4);
             graphics.dispose();
 
-            // finally save the chart
-            final byte[] bytes = new PngEncoder(bufferedImage, false, 0, pngCompressionLevel).pngEncode();
-            org.apache.commons.io.FileUtils.writeByteArrayToFile(outputFile, bytes);
+            // Encode image as webp using default settings and save it as webp file
+            ImageWriter writer = ImageIO.getImageWritersByMIMEType("image/webp").next();
+         
+            // Set parameters for lossless webp files
+            WebPWriteParam writeParam = new WebPWriteParam(writer.getLocale());
+
+            // Notify encoder to consider WebPWriteParams
+            writeParam.setCompressionMode(ImageWriteParam.MODE_EXPLICIT);
+
+            // Set lossless compression
+            writeParam.setCompressionType(writeParam.getCompressionTypes()[WebPWriteParam.LOSSLESS_COMPRESSION]);
+         
+            // Set quality of images
+            writeParam.setCompressionQuality(webpCompressionQuality);
+
+            // Save the image
+            writer.setOutput(new FileImageOutputStream(outputFile));
```

Comment:
We need to make sure that all parent directories of `outputFile` exist, i.e. we need to call  something like `outputFile.getParentFile().mkdirs()` before. Seems like the PNG encoder did that internally, but ImageIO does not. Hence, some images failed to be created. See `xlt/log/reportgenerator.log`.

Reply:
Done